### DISCUSSION TITLE
Defining custom S3 endpoint

### DISCUSSION THREAD

**Discussion by dsaintilma-flinks** [IBM/core-dump-handler#57] [General]
Hello,

It's not clear to me while reading the doc & helm chart how to configure a custom S3 endpoint (ex: https://storage.googleapis.com for GCS buckets)

Thanks.

**Comment by No9** [comment]
Hey @dsaintilma-flinks  Thanks for your interest in the project.
This is a great question and I've put the response in the FAQ.md as part of the next release.

https://github.com/IBM/core-dump-handler/blob/pre-7.0.0/FAQ.md#how-do-i-use-the-custom-endpoint 

It would be great if you could let me know if that is clear enough and works for you.

**Reply by dsaintilma-flinks** [reply]
I've actually figured it out on my own and my setup is working.

```
  extraEnvVars: |-
    - name: S3_ENDPOINT 
      value: "https://storage.googleapis.com"
```

Thanks for sharing the link to the FAQ!

**Reply by No9** [reply]
Great - I've just pushed 7.0.0 which has some important memory fixes in that you should pick up.
